packages/client-mcp/src/rocketride_mcp/resources.py

```
@@ -0,0 +1,163 @@
+    if client is None:
+        return json.dumps({'pipelines': [], 'error': 'Client is not connected'})
+    try:
+        tasks = await get_tools(client)
+        pipelines: List[Dict[str, Any]] = []
+        for task in tasks:
+            pipelines.append(
+                {
+                    'name': task.get('name'),
+                    'description': task.get('description'),
+                }
+            )
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)